.github/workflows/mysql-integration.yaml

```diff
@@ -10,17 +10,16 @@ jobs:
     strategy:
       matrix:
         go: [1.16]
-        java: [11, 17]
+        java: [17]
```

Comment:
if we're going to drop one of the java versions to test with, i'd suggest dropping 17 for now.